content/en/blog/_posts/Use containerd to handle k8s.gcr.io deprecation

```diff
@@ -0,0 +1,174 @@
+---
+layout: blog
+title: "Use containerd to handle k8s.gcr.io deprecation"
+date: 
+slug: handle-registry-deprecation-w-containerd
+---
+
+# Use containerd to handle k8s.gcr.io deprecation
+
+The Kubernetes community is getting ready for yet another major change. Until fall of 2022, `k8s.gcr.io` container registry hosted many Kubernetes community-managed containers images like Cluster Autoscaler, metrics-server, cluster-proportional-autoscaler. The “gcr.io” is Google Cloud Registry. In order to be vendor neutral, the community is moving away from using Google Cloud Registry to host container images.
+
+As a result, Starting March 20th, traffic from the old `k8s.gcr.io` registry is being redirected to `registry.k8s.io`. The older `k8s.gcr.io` will remain functioning for sometime but it is eventually getting deprecated.
+
+![Image.jpeg](https://res.craft.do/user/full/9d54cc03-adfe-f72f-3389-565eb7356d1d/doc/13FE78CC-1DDD-47D9-B875-EE3FB15F916E/FC421666-8104-4CD9-8C7F-8E4F4BCE2B83_2/6olkWYEK1kl10toAonqYL40P1ZSQIUY2N6iHRwId78kz/Image.jpeg)
```

Comment:
1. Set an alt attribute that helps accessibility
2. Use SVG (or PNG if necessary). JPEG and friends are for photos.
3. Include the image in the commit itself. Don't reference an external image.

Reply:
Images are now included in the commit and are in PNG format.